src/pages/[platform]/build-a-backend/auth/using-sdk/index.mdx

```diff
@@ -0,0 +1,59 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Use AWS SDK for Swift',
```

Comment:
This is meant to be a multi-platform page eventually, right? We shouldn't mention Swift if that's the case.